README.md

```diff
@@ -1,2 +1,36 @@
-# functions-framework
-functions-frameworks overview
+# OpenFunction Functions Framework
+OpenFunction Functions Framework is designed to provide users of FaaS with a suite of tools to convert user code into application that can be adapted to multiple runtimes.
```

Comment:
to convert user code into application that can be adapted to multiple runtimes. =>
to convert user code into application code that supports multiple runtimes.